magenta/models/shared/melody_rnn_generate.py

```diff
@@ -125,13 +126,21 @@ def run_with_flags(melody_rnn_sequence_generator):
   if not os.path.exists(FLAGS.output_dir):
     os.makedirs(FLAGS.output_dir)
 
+
+#primer_sequence from written melody or given midi file or random note between midi #48 = 4C to midi #84 = 7C
```

Comment:
Ah okay so we do still generate a random primer when no primer is given. We just forgot about that case when we compute bpm.